frontend/src/common/models.ts

```diff
@@ -72,8 +72,8 @@ export interface IHousingCompanyWritable {
     developer: {id: string};
     property_manager: {id: string};
     acquisition_price: {
-        initial: number | null;
-        realized: number | null;
+        initial: number;
+        realized: number;
```

Comment:
`acquisition_price.initial` change is ok.
`acquisition_price.realized` should be nullable, there may be cases where the company is not yet finished, so the realizer price is unknown.

Reply:
Had a chat about the acquisition prices with Tomi prior to making these changes. These two values will change to `acquisition_price` and `summary.realized_acquisition_price` anyway, once those values are provided by the API  - as now the realized one is not the calculated sum of the prices of all apartments for the housing company in question.
I also changed the initial values to be 0 instead of null in the blank `formData` contents which the creation form defaults to, so that the comparison wouldn't cause errors (and so that the details page wouldn't display "null" as a value - as the real realized value for a company with no apartments is 0.
Should this still be allowed to be null?

Reply:
..or well, anyway I added the `null` option back to the model now, for `acquisition_price.realized`, just in case it would present some problems which I can't foresee if it's not there. It's not like it will cause errors in the front either way. :)